src/Autofac.Integration.ServiceFabric/Autofac.Integration.ServiceFabric.csproj

```diff
@@ -2,8 +2,9 @@
 
   <PropertyGroup>
     <Description>Autofac integration for Azure Service Fabric. Provides service factory implementations for Actors, Stateful Services and Stateless Services.</Description>
-    <VersionPrefix>2.2.0</VersionPrefix>
-    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
+    <VersionPrefix>2.3.0</VersionPrefix>
+    <TargetFrameworks>netstandard2.1;netstandard2.0;net461</TargetFrameworks>
```

Comment:
Service Fabric appears to still support all the way back to `net45`. I wonder if we will get any complaints. 😄

Reply:
Since Autofac 5.0 dropped the support for net45 it should be fine 😄